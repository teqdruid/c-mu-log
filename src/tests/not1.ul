wall(10);

wall($y) {
    4 < $y;
    $y < 6;
}

main() {AND:
    @print($y);
    {AND:
	$y > 1;
	$y < 15;
    }
!$e.wall(4,5);
    !wall($y);
}
