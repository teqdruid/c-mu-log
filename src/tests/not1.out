11..15

 ^^^  Solution  ^^^ 

1..4

 ^^^  Solution  ^^^ 

6..9

 ^^^  Solution  ^^^ 

No more solutions
